Makefile

```diff
@@ -218,6 +219,12 @@ image-push-helm:
 image-push-helm-multiarch:
 	./hack/image/push-manifest-list.sh $(HELM_IMAGE) ${HELM_ARCHES}
 
+image-build-sdk: build/operator-sdk-dev-linux-gnu
+	./hack/image/build-sdk-image.sh $(OPERATOR_SDK_IMAGE):dev
+
+image-push-sdk:
+	./hack/image/push-image-tags.sh $(OPERATOR_SDK_IMAGE):dev $(OPERATOR_SDK_IMAGE)
+
```

Comment:
I think we also need an `image-push-sdk-multiarch` target.

Reply:
Can you elaborate on this? I figured we only needed amd64 since we are building FROM ubi-8. Do we need to support other bases as well?

Reply:
ubi8 is already a multiarch manifest so it should "just work" with all of the architectures we care about. I think we can just mimic what we do for the helm and ansible images.